### PR TITLE
Add missing instances for Conway era

### PR DESCRIPTION
**Pull request by lehins**
This are all instances that were needed for consensus integration of ConwayEra and are implemented by @nfrisby

### CODE REVIEW

File: eras/conway/impl/src/Cardano/Ledger/Conway/Translation.hs [hunk 1]
```diff
@@ -93,7 +93,7 @@ instance Crypto c => TranslateEra (ConwayEra c) ShelleyGenesis where
           API.sgUpdateQuorum = API.sgUpdateQuorum genesis,
           API.sgMaxLovelaceSupply = API.sgMaxLovelaceSupply genesis,
           API.sgProtocolParams = translateEra' ctxt (API.sgProtocolParams genesis),
-          API.sgGenDelegs = API.sgGenDelegs genesis,
+          API.sgGenDelegs = API.sgGenDelegs genesis,    -- TODO Is this correct?
```
Comment (lehins) [inline eras/conway/impl/src/Cardano/Ledger/Conway/Translation.hs]: @JaredCorduan Could you please comment on this. I haven't fully groked what ConwayEra introduces yet.

Comment (JaredCorduan) [reply eras/conway/impl/src/Cardano/Ledger/Conway/Translation.hs]: The conway era will probably only provide the master key rotation. For mainnet, and other networks that start with the Byron era, I do not know of any need to modify the master keys inside the Shelley genesis, but I am not an expert on everything that the consensus layer uses the various ledger state for. It's probably true that shelley-only networks do need to modify `sgGenDelegs`. I made #2988 for exactly these questions.

**Review by JaredCorduan — APPROVED** [review]
Looks good to me! (but I am happy to give @nfrisby a chance to look this over before we merge it)

File: eras/conway/impl/src/Cardano/Ledger/Conway/Translation.hs (hunk 1 above)
Comment (lehins) [reply eras/conway/impl/src/Cardano/Ledger/Conway/Translation.hs]: Ok, I removed the TODO. If we need to do anything about it, we can do it in a follow up PR

Comment (JaredCorduan) [reply eras/conway/impl/src/Cardano/Ledger/Conway/Translation.hs]: perfect, thanks

**Pull request merged**